app/common/model/src/main/java/io/syndesis/common/model/integration/IntegrationOverview.java

```diff
@@ -60,6 +61,10 @@ default IntegrationBulletinBoard getBoard() {
 
     Optional<String> getUrl();
 
+    Optional<String> getUrlTo3scale();
```

Comment:
Perhaps

```suggestion
    Optional<String> getManagementUrl();
```

This way it's not tied to 3scale and we can point to the OpenShift console route if we'd like to.